zwave_js_server/model/value.py

```diff
@@ -195,7 +192,7 @@ def cc_version(self) -> int:
     @property
     def endpoint(self) -> Optional[int]:
         """Return endpoint."""
-        return self.data.get("endpoint")
+        return self.data.get("endpoint", 0)
```

Comment:
When is the endpoint missing in the Value data?

Reply:
practically speaking, I don't know. The docs implied that there are scenarios where `endpoint` is not provided, and when that is the case, the value is for the root device (0). If we are contending that endpoint is always available, we shouldn't be using a `get` here in the first place

Reply:
I understood @AlCalzone as it's ok to not provide the endpoint when contructing the ValueId property to use with the zwave-js library. It means the same thing as providing endpoint 0. But we will always get the endpoint back from the library in the state.

It would be good to have this verified.

Reply:
I'd like to move forward with this PR since it's causing issues for users. I will revert this change and we can come back to this separately